misk/web/@misk/common/webpack.config.js

```diff
@@ -1,6 +1,8 @@
 const path = require("path")
 const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer")
 
+const bundleAnalyzer = false
```

Comment:
Do we want these to be shared from one spot?

Reply:
I don't think so at this point since you'd only need to use this to debug the compiled code size per package. So, if you wanted to debug `@misk/components` being > 5mb, you'd only want to turn it on for `@misk/components` not for all `@misk/` packages since their sizes don't contribute to the problem.